cypress/e2e/attribute_validations/reporting.cy.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from failing the test
+  return false
+})
+
+describe('Add multiple clusters and search', () => {
+  // This will hold the fixture data and the flag for skipping the tests
+  let skipTest = false
+
+  before(function () {
+    // Load the fixture data before running any tests
+    cy.fixture('data').then((data) => {
+      this.data = data
+      // If mode is "matter", set skipTest flag to true
+      if (this.data.mode === 'matter') {
+        skipTest = true
+      }
+    })
+  })
+
+  it('create two endpoints and validate basic information', function () {
+    // Skip the test if the flag is true (mode is "matter")
+    if (skipTest) {
+      cy.log('Skipping test because mode is "matter"')
+      return // Skip the test if mode is 'matter'
+    }
+
+    // Continue with the test if the mode is not "matter"
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.setZclProperties()
+
+    // Create and validate the first endpoint
+    cy.fixture('data').then((data) => {
+      cy.addEndpoint(data.endpoint4, data.cluster1)
```

Comment:
I do not understand this as I am not familiar with what is happening here. Comment says first endpoint and `cy.addEndpoint(data.endpoint4, data.cluster1)` says data.endpoint4

Reply:
yes, this is just how cypress is setup for all tests but I agree it should be fixed and cleaned up. I will work on a follow up PR to fix Cypress in general